apps/nextjs-website/src/app/theme.ts

```diff
@@ -11,6 +11,8 @@ declare module '@mui/material/styles' {
       readonly primaryColorDark: string;
       readonly backgroundColorDefault: string;
       readonly darkLinkColor: string;
+      readonly grayLinkColor: string;
+      readonly blackOpacized: string;
```

Comment:
```suggestion
      readonly black50: string;
```
"Opacized" isn't really a word. I would include the opacity percentage in the variable's name.